configs/webpack.config.base.js

```diff
@@ -62,7 +35,7 @@ export default {
    */
   resolve: {
     extensions: ['.js', '.jsx', '.json'],
-    modules: [path.join(__dirname, '..', 'app'), 'node_modules']
+    modules: [path.join(__dirname, '..'), 'node_modules']
```

Comment:
I think we can just remove this whole line. I believe it was originally meant to be `path.join(__dirname, '..', 'app/node_modules')`. Was changed during the single package.json setup in #1854.